introduction.md

```diff
@@ -0,0 +1,4 @@
+* Git
+* iOS
+* Ruby
+* CoffeeScript
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?